locales/pt_BR/pt_BR.json

```diff
@@ -633,7 +633,7 @@
     "Tonga": "Tonga",
     "Too Many Requests": "Muitas solicitações",
     "total": "total",
-    "Total:": "Total:",
+    "Total:": "total:",
```

Comment:
Why was it changed to lower case?

Reply:
If the spelling of the word matches the English version, then it must be specified in the [exclusion](https://github.com/Laravel-Lang/lang/blob/master/excludes/pt_BR.php) file.